sem/runner.py

```diff
@@ -7,7 +7,9 @@
 import sem.utils
 
 from tqdm import tqdm
+from typing import Final
 
+SIGKILL_CODE: Final = -9    # POSIX return code which usually corresponds to out of memory events.
```

Comment:
Is it necessary to import typing.Final here? We don't use typing anywhere else in the project.

Reply:
It is not necessary, it just seemed "good practice" to use it for marking the variable (almost) as a constant. At least, that was my understanding from the PEP guideline which I linked above. If you prefer to limit the imports though I can remove it!